rift/node.py

```diff
@@ -1543,22 +1551,27 @@ def get_table_param(cli_session, parameters):
                                   '"default", "unspecified", or number)'.format(table_str))
                 return None
 
-
-
     def command_show_spf_dir(self, cli_session, parameters):
-        direction = self.get_direction_param(cli_session, parameters)
+        (direction, special_for_neg_disagg) = self.get_direction_param(cli_session, parameters)
+
         if direction is None:
             return
-        self.command_show_spf_destinations(cli_session, direction)
+
+        self.command_show_spf_destinations(cli_session, direction, special_for_neg_disagg=special_for_neg_disagg)
```

Comment:
special_for_neg_disagg=special_for_neg_disagg => special_for_neg_disagg

just as clear and less verbose